src/js/cart.js

```diff
@@ -0,0 +1,32 @@
+import { GET_PRODUCT } from "./constant.js";
+
+async function getToCart() {
+  try {
+    const localCart = JSON.parse(localStorage.getItem("cart"));
```

Comment:
```suggestion
    const localCart = JSON.parse(localStorage.getItem("cart")) || [];
```
Bổ sung thêm default value nếu key cart k tồn tại nữa nhé. Ngoải ra thì có thể tách ra 1 hàm riêng để dùng để get localstorage, tránh bị lặp code